packages/aws-cdk-lib/aws-ecs-patterns/README.md

```diff
@@ -108,9 +110,11 @@ const loadBalancedFargateService = new ecsPatterns.ApplicationMultipleTargetGrou
   taskImageOptions: {
     image: ecs.ContainerImage.fromRegistry("amazon/amazon-ecs-sample"),
   },
+  minHealthyPercent: 33,
   targetGroups: [
     {
       containerPort: 80,
+      deregistrationDelay: 300
```

Comment:
```suggestion
      deregistrationDelay: Duration.seconds(300),
```

Reply:
Apologies, I couldn't get the tests running on my machine even before changes. Various things complained and/or timed out.

Updated with your corrections.

Reply:
FWIW confirmed the change itself works by using `patch-package`, though obviously understand we need the tests to pass.